src/ripple/core/impl/Config.cpp

```diff
@@ -411,26 +413,22 @@ void Config::loadFromString (std::string const& fileContents)
     if (getSingleSection (secConfig, SECTION_SSL_VERIFY, strTemp, j_))
         SSL_VERIFY          = beast::lexicalCastThrow <bool> (strTemp);
 
-    if (getSingleSection (secConfig, SECTION_VALIDATION_SEED, strTemp, j_))
+    if (getSingleSection (secConfig, SECTION_VALIDATION_SEED, VALIDATION_SEED, j_))
```

Comment:
If `getSingleSelction` returns false clear `VALIDATION_SEED`. (We could also consider doing this inside of `getSingleSection`, but we'd have to audit existing code first).